src/history/RepositoryViewDelegate.cpp

```diff
@@ -515,26 +515,25 @@ void RepositoryViewDelegate::paintTagBranch(QPainter *painter, QStyleOptionViewI
          o.font.setBold(isCurrentSpot);
 
          const auto nameToDisplay = showMinimal ? QString(". . .") : *mapIt;
-         QFontMetrics fm(o.font);
+         const QFontMetrics fm(o.font);
          const auto textBoundingRect = fm.boundingRect(nameToDisplay);
          const int textPadding = 3;
          const auto rectWidth = textBoundingRect.width() + 2 * textPadding;
+         const QRectF markerRect(o.rect.x() + startPoint, o.rect.y() + 2, rectWidth, ROW_HEIGHT - 4);
```

Comment:
This changes make the padding bigger what is unrelated with the text position.

Reply:
Yes, because the padding is not sufficient to contain the text. The text goes outside the rectangle on both Linux and Windows, don't you see that on your system?
![Screenshot_20210312_210112](https://user-images.githubusercontent.com/78597079/110992767-aaceef00-8376-11eb-8565-f6667e4ca8c8.png)

Reply:
I don't see that at all. And I think that [other users don't see that](https://github.com/francescmm/GitQlient/issues/194), so might be some system font configuration.

Do you have font at a different size than 100% in the system?

Reply:
I have DPI set to 120% on both systems. Maybe you test only with DPI=100?

Reply:
That's correct. Initially Qt supports (and in GitQlient is set on) the HiDPI. However, I think there is a problem in the delegates and Qt is not calculating the new coordinates as it should.

I think I could borrow a 4K screen to test.

Reply:
You don't need 4K screen to test high DPI modes. You can set DPI to 120 or 150 on Windows and Linux regardless of screen resolution. My screen is Full HD.

Reply:
But anyway with my fix it looks good on DPI 96 and 120